File: testdata/bench/csharp/BenchmarkTest00001.cs
using System.Data.SqlClient;
using Microsoft.AspNetCore.Mvc;

namespace BenchApp.Controllers
{
    public class SqliController : Controller
    {
        [HttpGet]
        public IActionResult Search()
        {
            string input = Request.Query["name"];
            var conn = new SqlConnection("Server=localhost;Database=app;");
            conn.Open();
            var cmd = new SqlCommand("SELECT * FROM users WHERE name = '" + input + "'", conn);
            var reader = cmd.ExecuteReader();
            return Ok();
        }
    }
}

File: testdata/bench/lua/BenchmarkTest00001.lua
local mysql = require "resty.mysql"
local args = ngx.req.get_uri_args()
local name = args.name
local db = mysql:new()
db:connect({host="127.0.0.1", port=3306, database="app"})
local res = db:query("SELECT * FROM users WHERE name = '" .. name .. "'")
ngx.say(res)
